simulation/sub8_gazebo/nodes/pinger_sim.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+import rospy
+from gazebo_msgs.srv import GetModelState
+from sub8_msgs.srv import Sonar
+from sub8_sonar import EchoLocator
+from sub8_ros_tools import msg_helpers
+from sub8_ros_tools.geometry_helpers import rotate_vect_by_quat
+import tf
+
+import numpy as np
+
+
+class Pinger():
+    def __init__(self, wave_propagation_speed, precision=14):
+        self.get_model = rospy.ServiceProxy('/gazebo/get_model_state', GetModelState)
+
+        self.wave_propagation_speed = wave_propagation_speed
+        self.precision = precision
+
+        self.hydrophone_board_tf = np.array([0, 0, 0])
+        self.hydro_loc =  {'hydro0': {'x': 0,'y': 0,'z': 0},
```

Comment:
The sonar launch never gets launched for the sim right? That would mean we would have to change it in both launch files regardless.